nucliadb/nucliadb/train/generators/question_answer_streaming.py

```diff
@@ -106,17 +105,23 @@ async def iter_stream_items(
     question_answer_pb: QuestionAnswer,
 ) -> AsyncGenerator[QuestionAnswerStreamItem, None]:
     question_pb = question_answer_pb.question
+    question_paragraphs = [
+        await get_paragraph(kbid, paragraph_id)
```

Comment:
just one thought: do we want to try/catch not found errors on the get_paragraph just in case the annotation points to a wrong paragraph id?

Reply:
Sure, I'll change it